docs/postgres.md

```diff
@@ -9,7 +9,7 @@ describes how it can be configured.
 To build LND with postgres support, include the following build tag:
 
 ```shell
-⛰  make tags="kvdb_postgres"
+⛰  make extratags="kvdb_postgres"
```

Comment:
This will now also indirectly add the `dev` tag. Should we mention that here? E.g. "To create a dev build of lnd with postgres support, ..."